packages/flame_tiled/lib/src/tile_atlas.dart

```diff
@@ -156,4 +156,10 @@ class TiledAtlas {
       key: key,
     );
   }
+
+  /// If you called `Flame.images.clearCache()` you also need to call this
```

Comment:
The first line of the dartdoc should explain what the function does, and then this can come afterwards.